src/main/java/pl/zaradny/springApp/domain/Description.java

```diff
@@ -0,0 +1,19 @@
+package pl.zaradny.springApp.domain;
+
+public final class Description {
+
+    private final String text;
+
+    public Description(String text) {
+        this.text = text;
+    }
+
+    public Description(DescriptionDto descriptionDto) {
```

Comment:
to wiązanie jest brzydkie, go wiąże ci obiekt transportowy z domenowym ;) lepiej zostać przy konstruktorze na podstawie String text